webviz_config/generic_plugins/_example_wlf_plugin/_plugin.py

```diff
@@ -0,0 +1,114 @@
+from typing import List
+
+from dash import Input
+
+from webviz_config import WebvizPluginABC
+from webviz_config.utils import StrEnum
+from .views._plot import PlotView, PlotViewElement, PlotViewSettingsGroup
+from .views._table import TableView, TableViewElement, TableViewSettingsGroup
```

Comment:
I disagree. I like to import from the explicit view folder and not everything from `views`.

Reply:
Not a very big detail - so fine for me.

My idea was not to import everything from `_views`, but importing views to a plugin using `from ._views import PlotView, TableView`. Thereby the views are exposed to the users, but the content of the views, i.e. `ViewElement` and `ViewSettingsGroup` must be accessed using direct path. This would give the following:

```
from ._views import PlotView, TableView
from ._views._plot import PlotViewElement, PlotViewSettingsGroup
from ._views._table import TableViewElement, TableViewSettingsGroup
```